### PR TITLE
add compat data for SVGLineElement

### PR DESCRIPTION
**Pull request by solfen**
Link: https://developer.mozilla.org/en-US/docs/Web/API/SVGLineElement

### CODE REVIEW

File: api/SVGLineElement.json [hunk 1]
```diff
@@ -0,0 +1,240 @@
+{
+  "api": {
+    "SVGImageElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGImageElement",
```
Comment (maboa) [inline api/SVGLineElement.json]: I think this should be `https://developer.mozilla.org/docs/Web/API/SVGLineElement`

File: api/SVGLineElement.json [hunk 2]
```diff
@@ -0,0 +1,240 @@
+{
+  "api": {
+    "SVGImageElement": {
```
Comment (maboa) [inline api/SVGLineElement.json]: `"SVGLineElement"` I think.

File: api/SVGLineElement.json [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGImageElement",
+        "support": {
+          "webview_android": {
+            "version_added": false
```
Comment (jpmedley) [inline api/SVGLineElement.json]: `true`

File: api/SVGLineElement.json [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: `true` for all 3 flavors of Chrome.

File: api/SVGLineElement.json [hunk 5]
```diff
@@ -0,0 +1,240 @@
...
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: `true` for both.

File: api/SVGLineElement.json [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: ditto

File: api/SVGLineElement.json [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: ditto

File: api/SVGLineElement.json [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: ditto

File: api/SVGLineElement.json [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: ditto

File: api/SVGLineElement.json [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+        "__compat": {
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: ditto

File: api/SVGLineElement.json [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/SVGLineElement.json]: ditto

**Pull request merged**